FlorenceBML/BMLor403/BMLor403.xml

```diff
@@ -34,8 +35,421 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <altIdentifier>
                             <idno>Marrassini ms. 11</idno>
                         </altIdentifier>
+                        <altIdentifier>
+                            <idno>BML-011</idno>
+                        </altIdentifier>
                     </msIdentifier>
+                    <msContents>
+                        <summary/>
+                            <msItem xml:id="ms_i1">
+                                <locus from="1r" to="47v"/>
+                                <title ref="LIT1201Bartos"/>
+                                <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ። ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፡ ዛቲ፡ ጸሎት። እግዚእትነ፡ ማርያም፡ ዘጸለየት፡ ባቲ፡ 
+                                    በውስተ፡ ሀገረ፡ ባርቶስ፡ ወተፈትሐ፡ ኵሉ፡ ኅጻውንት፡ ወአድኀነቶ፡ ለማቲያስ፡ ረድእ፡ ወእምኑ፡ ሰብእ፡ ሀገር፡ ጸሎታ፡ የሀሉ፡ ምስሌነ፡ አሜን።</incipit>
+                                <explicit xml:lang="gez">ዘንተ፡ ይቤ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ሎቱ፡ ስብሐት። ወለወላዲቱ፡ ድንግል፡ ወወሀባ፡ ሰላመ፡ ወዐርገ፡ ውስተ፡ ሰማያት፡
+                                    በዐቢይ፡ ስብሐት፡ ወትረ፡ እስከ፡ ለዓለመ፡ ዓለም፡ አሜን። አሜን። ወአሜን።</explicit>
+                                <note>The name of the owner <ref target="PRS0000Simon">ስምዖን</ref> has been inscribed 
```

Comment:
it would be best to move this information to additions as it does not belong to the content (text) but to the paracontent (physDesc, creation of the object)

Reply:
I am not sure, whether or not the name of the owner is a later addition. We need to take into account, that the name was frequently introduced by gabra-ka "your servant (Simon)". Even if the name was inserted in red it may have been the plan to insert him right from the beginning.

Reply:
If I move it to the additions, I suppose it is enough to make a more general indication as "Name of the owner Simon occurs varios times in most textparts." and delete the notes in the msItem-records.

Reply:
Done, as you requested.

Reply:
We had lengthy discussions on supplications and similar insertions by the original hand. While indeed these are not true additions the solution we finally agreed to live with is that these, unless the whole text is encoded and marked up correctly, go to extras
https://github.com/BetaMasaheft/Documentation/issues/895

Reply:
I understand. I will redefine it as an extra (e1) instead of addition (a1). I will try to recover the loci as well.

Reply:
I have done it both. Indeed it is not a big deal to move the already encoded information from one spot to another. I understand the reasons to arrange it like this and try to be consistent in any case.